docs/fieldnorm/control_of_documents.md

```diff
@@ -1,25 +1,92 @@
+
+## Candidates for a Bioinformatician friendly QMS
+
+_Bioinformaticians_ create workflows collaboratively on a computer within a git environment. Git is a distributed version control system that tracks changes in any set of computer files. This can be used to track the changes of the code itself, but also of its usage in Standard Operating Procedures (SOPs). Git comes in with a set of handy features, allowing to track issues and their resolution, or to identify each new release of a workflow and its newest features for example. As workflows often get upgraded and updated, their usage in SOP is susceptible to frequent changes as well. Examples of those documentations systems are:
 
 1. Github pages
-2. Gitlab
-3. Confluence
\ No newline at end of file
+2. Gitlab pages
+3. Confluence
+
+Many of those tools, if used in a specific way, can allow _Bioinformaticians_ to work in a comfortable environement, while complying to the ISO norms requirements. They are of course multiple ways to implement such a system, and as long as it is documented how your system complies to the ISO-15189 requirements, all is well. If you are looking for some inspirations, in the next paragraph we will describe an example on how to use one this tool to follow the ISO requirements for both controlling the workflows/software you develop, and their associated documentation/SOP.
+
+## Example case : The usage of GitLab
+
+### Central gathering of all SOP on Gitlab pages
+
+ Here we provide an example how GitLab can be used as a _Bioinformaticians_ QMS. Using gitlab pages and mkdocs, you can easily create a well structured _documentation_ to store all your SOPs . Here are some guideline to illustrates the point of attention to comply with ISO-15189 :
+
+1. Each SOP becomes a markdown file. Its name will include a number for easy and unique identification of the document. The numbering can also be used to classify each SOP into categories. _Example_ :
+   
+- 10 - Data Access
+    - 11 - ExternalSequencers -> 11_externalsequencers.md
+    - 12 - InternalSequencers -> 12_internalsequencers.md
+- 20 - Data Processing
+    - 21 - WES -> 21_wes_flow.md
+    - 22 - RNAseq -> 22_rnaseq_flow.md
+- 30 - Data sharing
+    - 31 - Cluster -> 31_share_cluster_user.md
+    - 32 - External -> 32_share_external_user.md      
+
+ Here we seperate 3 flows in categories : accessing data, processing data and sharing data, allowing intuitive navigation within your documentation depending on the action you are trying to perform.
+
+2. The repository is set with special users type permissions and structure :
+    - In a git way, a __"main"__ branch corresponds to the __"production"__ _documentation_, and a __"dev"__ branch mirrors the main branch at first, and will then accumulates modifications that needs to appear in the upcoming release of the _documentation_
+    - 🤵 __Members__ can create issues to report something incorrect with a specific document, or the need to create a new document
```

Comment:
I am still not very fond of the emoji's in the text, but we can discuss it on Friday :slightly_smiling_face: as I also saw it in other contributions.

Reply:
Sorry I completely missed this Friday's meeting :'( What was the decision about the emojis?

Reply:
Complete removal. Emojis are rendered differently depending on the software. There are many competing emoji picture sets which can differ in many key areas. So for consistency it was chosen to drop them.

Reply:
About the original comment : As a more general comment not specific to this document, the style in this document is addressing the reader directly with "you" and the writers as "we". Which is okay, but it is a style difference to some of the other documents. We should discuss these sort of overarching style issues on Friday.

So what is the stance I should adopt here you think ? Did we agreed on "we" or on "you" ?

Reply:
We did not talk about it. So let's merge it like this.